data/lib/core/player.lua

```diff
@@ -312,3 +312,44 @@ function Player.getWeaponType(self)
 	end
 	return WEAPON_NONE
 end
+
+local function prepareUpdateKillTrackerMsg(monster, corpse)
+	monsterType = monster:getType()
+	if monsterType then
+		return nil
+	end
```

Comment:
Should monsterType be global? Did you forget to add `local`?
And the conditional statement looks weird, If monsterType is set, return nil? Did you forget to add `not`?

Reply:
correct, should be local